WcaOnRails/app/views/git_hub/update_translation.html.erb

```diff
@@ -6,7 +6,7 @@
     <div class="panel-body">
       <p>You can find the corresponding Pull Request <%= link_to "on GitHub", @pr_url %>.</p>
       <p>This sweet cat GIF is for you =)</p>
-      <a href="http://thecatapi.com"><img src="http://thecatapi.com/api/images/get?format=src&type=gif"></a>
+      <a href="http://thecatapi.com"><img src="https://thecatapi.com/api/images/get?format=src&type=gif"></a>
```

Comment:
Since they support it, how about linking to https://thecatapi.com/ in the href as well?